src/frontend/index.tsx

```diff
@@ -44,7 +44,7 @@ export let intl: IntlShape;
 // Wait for the DOM to load before we scour it for an element that requires React to render
 document.addEventListener('DOMContentLoaded', async event => {
   try {
-    if (!Intl) {
+    if (!global.Intl) {
```

Comment:
Should this not be `window.Intl`? Who do we expect to set this `global` variable?

Reply:
This is the test they amde in the polyfille documentation https://github.com/andyearnshaw/Intl.js/#intljs-and-browserifywebpack

Reply:
I'm pretty sure this will not work.

They say `global` either as pseudocode (use your environment's global object — eg `Node`'s may differ) or because they think you'd wrap this in some sort of IIFE/UMD thing which passes `window` relabeled as `global`

Reply:
Ack i change to `window`